src/main/java/com/spotify/docker/client/messages/swarm/Service.java

```diff
@@ -71,10 +71,37 @@
     @Nullable
     public abstract String serviceName();
 
+    /**
+     * Filter by label.
+     */
+    @Nullable
+    public abstract Label label();
```

Comment:
Can a service really only have one label? All the other objects that have labels can have an arbitrary number of them.